src/client/dfuse/ops/read.c

```diff
@@ -511,17 +513,6 @@ chunk_read(fuse_req_t req, size_t len, off_t position, struct dfuse_obj_hdl *oh)
 				d_slab_release(cd->eqt->de_read_slab, cd->ev);
 				D_FREE(cd);
 			}
-		} else {
-			struct dfuse_info *dfuse_info = fuse_req_userdata(req);
-			struct dfuse_eq   *eqt;
-
-			eqt = pick_eqt(dfuse_info);
-
-			/* Send a message to the async thread to wake it up and poll for events */
-			sem_post(&eqt->de_sem);
-
-			/* Now ensure there are more descriptors for the next request */
-			d_slab_restock(eqt->de_read_slab);
```

Comment:
do you restock elsewhere?

Reply:
restock is actually not needed in this path, since no RPC is issued, so no buffer needed.